### PR TITLE
Service Page  Section

### LINKED ISSUE #24: [Feature]: Service page
image

Inspiration :- https://www.designesia.com/themes/buildpro/services.html

### PR DESCRIPTION
**Pull request by rolikaagarwal**
CLOSES #24 

Full Screen- 

image

Screen Size Reduced- 

image

Mobile Screen- 

image

### CODE REVIEW

**Comment by AbhiPatel10** [comment]
Make a separate service Page

**Comment by AbhiPatel10** [comment]
@rolikaagarwal Are you working on it?

**Comment by rolikaagarwal** [comment]
Yes I'm working

File: ServiceSection/style.css [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+     justify-content: space-between;
+     max-width: 90%;
+     margin: 0 auto;
+     padding: 15px;
+     box-sizing: border-box;
+   }
+  
+   .servicesection-image {
+     width: calc(33.33% - 20px);
+     margin-bottom: 20px;
```
Comment (AbhiPatel10) [inline ServiceSection/style.css:47]: margin: 20px 0px;

**Review by AbhiPatel10 — CHANGES_REQUESTED** [review]

**Comment by rolikaagarwal** [comment]
Changes Fixed.

**Pull request merged**